### DISCUSSION TITLE
[预览] v2.7.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4158] [分享 / Share]
包含 [v2.7.1](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.1) 的所有更新内容.

- Comments API 支持读取评论图片. (PR #4130 by [manip_ego](https://github.com/manipEgo))
```diff
export interface CommentItem extends CommentReplyItem {
+  /** 评论图片 */
+  pictures?: string[]
  /** 回复 */
  replies: CommentReplyItem[]
  // ...
}
```
- `显示视频投稿时间` 允许自定义替换文本格式. (PR #4136 by [呼乎户](https://github.com/wisokey))
- `清爽首页` 排行榜支持切换为列表模式, 能够查看 10 项左右的排行. (#3611, #3175)
- 新增功能 `动态看图自动回顶`. (#4029)
> 在动态里查看图片详情时 (非全屏), 切换图片自动回到图片顶部; 退出查看图片模式时, 自动将动态移入视图内.

- 重构了 `直播信息扩充` 面板, 改为完全自主实现, 支持显示超过 10 条的直播间, 并可以进行搜索和刷新操作. (#3774, #4035, #4034)

<img src="https://user-images.githubusercontent.com/26504152/234049176-7ae3e3cf-6956-4efe-aec1-66218d940c93.png" width="300">

- 新增功能 `反转滚轮调音量`. (#4053)
> 反转在网页全屏 / 全屏模式下使用滚轮调节音量的方向, 使其更符合使用触控板时的操作方向. 请注意不能和 `禁止滚轮调音量` 一同使用.
> - 手指向上推时, 增加音量
> - 手指向下推时, 减少音量
> - 可以自定义 `灵敏度`, 同样的滚动幅度下, 灵敏度越高变化的音量越多

- 新增功能 `自动点赞`. (#2354)
> 进入视频 / 查看动态时, 自动点赞.

- 新增功能 `隐藏用户信息卡片`. (PR #4093 by [WakelessSloth56](https://github.com/WakelessSloth56))
> 隐藏鼠标指向用户名或用户头像时弹出的浮动用户信息卡片.

- 新增功能 `播放器置顶（新）`. (#3030, #PR #4152 by [RieN 7z](https://github.com/rien7))
> 原来的播放器置顶组件，现在已经不可用了，这是一个新的版本，可以在视频页面中将播放器放在页面最上方.

- 新增插件 `搜索栏 - UID 跳转`, `搜索栏 - 数字联想`. (#677)
> 在输入 UID (用户 ID) 时, 提供对应的跳转选项. / 在输入纯数字时, 提供以下选项:
> - 跳转至相应的视频 (视为 av 号)
> - 跳转至相应的专栏 (视为 cv 号)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.1-preview'>[预览] v2.7.1</a>.</em>

**Comment by Geeyun-JY3** [comment]
搜索栏跳转对应的 issue 是 #677

**Reply by the1812** [reply]
手滑（
